datahub-frontend/app/auth/sso/oidc/OidcCallbackLogic.java

```diff
@@ -241,66 +242,79 @@ private CorpUserSnapshot extractUser(CorpuserUrn urn, CommonProfile profile) {
     final OidcConfigs configs = (OidcConfigs) _ssoManager.getSsoProvider().configs();
 
     // First, attempt to extract a list of groups from the profile, using the group name attribute config.
-    final String groupsClaimName = configs.getGroupsClaimName();
-    if (profile.containsAttribute(groupsClaimName)) {
-      try {
-        final List<CorpGroupSnapshot> groupSnapshots = new ArrayList<>();
-        final Collection<String> groupNames;
-        final Object groupAttribute = profile.getAttribute(groupsClaimName);
-        if (groupAttribute instanceof Collection) {
-          // List of group names
-          groupNames = (Collection<String>) profile.getAttribute(groupsClaimName, Collection.class);
-        } else if (groupAttribute instanceof String) {
-          // Single group name
-          groupNames = Collections.singleton(profile.getAttribute(groupsClaimName, String.class));
-        } else {
-          log.error(String.format("Failed to parse OIDC group claim with name %s. Unknown type %s provided.",
-              groupsClaimName,
-              groupAttribute.getClass()));
-          // Return empty list. Do not throw.
-          return Collections.emptyList();
-        }
+    //final String groupsClaimName = configs.getGroupsClaimName();
+    final List<CorpGroupSnapshot> extractedGroupNames = new ArrayList<>();
```

Comment:
Why is this named `extractedGroupNames`? It is a list of snapshots, not names... Can we rename to `extractedGroups`?

Reply:
Sure thing